index.template.html

```diff
@@ -405,13 +405,13 @@ <h2>Thing Description</h2>
   <p>
   The WoT Thing Description (TD) is a central building block in the W3C Web of
   Things (WoT) and can be considered as the entry point of a <a>Thing</a>
-  (much like the <i>index.html</i> of a Web site). A TD instance has four main
+  (much like the <i>index.html</i> of a Web site). A TD instance has five main
   components: textual metadata about the <a href="#thing">Thing</a> itself,
   a set of <a href="#interactionaffordance">Interaction Affordances</a>
   that indicate how the <a>Thing</a> can be used,
   <a href="#sec-data-schema-vocabulary-definition">schemas</a> for the data
-  exchanged with the <a>Thing</a> for machine-understandability,
-  and, finally, <a href="#sec-hypermedia-vocabulary-definition">Web links</a> to 
+  exchanged with the <a>Thing</a> for machine-understandability, <a href="#sec-security-vocabulary-definition">Security Definitions</a>
+  provide metadata about the security mode of the <a>Thing</a> that must be used for secure interaction, and, finally, <a href="#sec-hypermedia-vocabulary-definition">Web links</a> to 
```

Comment:
to provide 

the previous sentences have this kind of sentence structure

Reply:
Another point:
about the security mechanisms that must be used for interactions

secure interaction is a bit weird for me but maybe just me

Reply:
done